@noun-auction/components/ActiveAuction/ActiveAuctionRow.tsx

```diff
@@ -117,14 +177,20 @@ export function ActiveAuctionRow({
       >
         {layout === 'sideBarBid' && (
           <AuctionHighBid
-            layoutDirection="column"
+            auctionCompleted={auctionCompleted}
+            layout={layout}
+            highestBid={highestBid}
+            collectionAddress={collectionAddress}
+            styles={{
+              layoutDirection: 'column',
```

Comment:
@olegakbarov watch out, layoutDirection isn't a CSS property. 'direction' would work, but looks like layoutDirection is a prop for use within AuctionHighBid. Might need to look at how it's used there.